plugins/org.jboss.reddeer.junit/src/org/jboss/reddeer/junit/requirement/Requirement.java

```diff
@@ -45,4 +47,19 @@
 	 * Clean up after requirement.
 	 */
 	void cleanUp();
+	
+	/**
+	 * Decides whether it is possible to accept specific requirement's configuration object or not
+	 * @return true, if declaration fits in
+	 */
+	default boolean isDeclarationAcceptable() { return true; }
+	
+	/* (non-Javadoc)
+	 * @see org.jboss.reddeer.junit.execution.IExecutionPriority#getPriority()
+	 */
+	@Override
+	default long getPriority() {
+		return 1;
```

Comment:
value could be 0, as the default one. So other requirement could choose from positive range (to be executed sooner) or negative range (to be executed later) in comparison to default implementation.